control-protocol.md

```diff
@@ -439,6 +470,16 @@ is intended.
   # or even more.
   "seq" : <uint64_t>
 
+  # Randomly picked measurement id, stable for one measurment. All
+  # subsequent requsts/replies to the particular message must use this measurment-id.
+  # Think about two parallel ongoing udp-goodput measurements. The client will
+  # alternating query info-reply messages for both ongoing measurements with
+  # both measurement-id's.
+  #
+  # The client dicatates the measurement id for one measurements. The server will
+  # reply this id in the reply message too.
+  "measurement-id" : "<uint64_t>"
+
   # to implement a trivial access mechanism a secret can be given.
```

Comment:
note: the "configuration" object within "measurement" is also already refined: including name/value pairs "type", "configuration", "name" that dont show up here. we could add that to the docs